GAE/src/org/waterforpeople/mapping/app/web/rest/UserRestService.java

```diff
@@ -110,7 +110,7 @@ private boolean isUserAdminRole(User user) {
     }
 
     private boolean isSuperAdminRole(User user) {
-        return (user.isSuperAdmin() != null && user.isSuperAdmin())
+        return (user.isSuperAdmin()
                 || user.getPermissionList()
                         .equals(Integer.toString(AppRole.SUPER_ADMIN.getLevel()));
```

Comment:
This exposure of the permissionList to determine the user role outside the User class is also unnecessary -- rather move this check back into the isSuperAdmin() getter as a refactoring.